### PR TITLE
Check folder name matches start of file

### LINKED ISSUE #17: verify_filenames.py doesn't check that files are in the right folder
Could potentially have 2019 minutes in the 2020 folder for example

### PR DESCRIPTION
**Pull request by tpoomlmly**
Fixes #17

### CODE REVIEW

File: verify_filenames.py [hunk 1]
```diff
@@ -63,21 +63,28 @@
 
     for f in folder.iterdir():
 
-       # Test if there are any nested folders
+        # Test if there are any nested folders
```
Comment (trickeydan) [inline verify_filenames.py:66]: Not sure why whitespace needs changing here?

File: verify_filenames.py [hunk 2]
```diff
@@ -63,21 +63,28 @@
...
 
-       if f.is_dir():
+        if f.is_dir():
```
Comment (trickeydan) [inline verify_filenames.py:68]: Same again re whitespace

File: verify_filenames.py [hunk 3]
```diff
@@ -63,21 +63,28 @@
...
            print(f"\tFound nested directory: {folder.stem}/{f.stem}")
            status = False
-
```
Comment (trickeydan) [inline verify_filenames.py:71]: Diff looks strange here. Has intellij messed with indentation?

**Review by trickeydan — COMMENTED** [review]
Logic is fine. Run a linter on the indentation?

File: verify_filenames.py (hunk 3 above)
Comment (tpoomlmly) [reply verify_filenames.py:71]: I was editing on github, so there was probably some extra whitespace I left in by accident

File: verify_filenames.py (hunk 1 above)
Comment (tpoomlmly) [reply verify_filenames.py:66]: that particular block was indented with 3 spaces instead of 4

File: verify_filenames.py (hunk 2 above)
Comment (tpoomlmly) [reply verify_filenames.py:68]: same again as above

File: verify_filenames.py (hunk 3 above)
Comment (tpoomlmly) [reply verify_filenames.py:71]: nevermind, it was the opposite - there was some trailing whitespace I removed

**Review by trickeydan — APPROVED** [review]

**Pull request merged**